src/actions/nodes.test.js

```diff
@@ -42,24 +43,35 @@ describe('node actions', () => {
       jest.resetModules();
     });
 
+    describe('if meta flag is false', () => {
+      it('should not make any API calls if meta is false', () => {
+        const store = createStore(reducer, mockState.json);
+        loadNodeData('parametersID')(store.dispatch, store.getState);
+        expect(store.getState().loading.node).toBe(false);
```

Comment:
It might be worth explicitly setting the flag to false here, so that the test will still make sense even if the default is changed later upstream.

Reply:
yea great point that you raised here - originally I thought of setting it to false here for the test could be more explicit, but given the default setting of meta is false and we have explicitly mentioned about the condition we are testing in this case, it doesn't really make sense to particularly set the flag to false here?

Reply:
Yeah I get that. My point was mainly that if someone changed the default upstream, the test might fail when it's not supposed to, or else if the test were broken then it might pass when it's not supposed to. But I'm not going to press the issue if you strongly feel that it shouldn't be set explicitly, because I understand your position!

Reply:
Yeah it doesn't look like the test guarantees that the flag is false, which is what the description implies, so it's probably worth ensuring that or changing the describe to handle the default case (which may be true or false).

Reply:
@studioswong ☝️